src/consensus/pbft/pbft.h

```diff
@@ -106,19 +108,22 @@ namespace pbft
     enclave::RPCSessions& rpcsessions;
     std::unique_ptr<consensus::LedgerEnclave> ledger;
     SeqNo global_commit_seqno;
+    std::unique_ptr<pbft::Store> store;
 
   public:
     Pbft(
```

Comment:
Minor: could we change the order of arguments to the Pbft constructor to be consistent with Raft, i.e.

```
std::unique_ptr<Store<kv::DeserialiseSuccess>> store,
      std::unique_ptr<LedgerProxy> ledger_,
      std::shared_ptr<ChannelProxy> channels_,
      NodeId id,
      std::chrono::milliseconds request_timeout_,  <<< Ignore for now
      std::chrono::milliseconds election_timeout_,  <<< Ignore for now
      bool public_only_ = false) <<< Ignore for now
```